15-eco.Rmd

```diff
@@ -25,44 +25,37 @@ library(vegan)             # community ecology package
+Fog oases, locally called *lomas*, are ecological systems found in the mountains along the coastal deserts of Peru and Chile.
+Similar ecosystems can be found elsewhere, including in the deserts of Namibia and along the coasts of Yemen and Oman [@galletti_land_2016].
+Extreme conditions and remoteness provide the habitat for a unique ecosystem, including species endemic to the fog oases.
+Despite the arid conditions and low levels of precipitation of around 30-50 mm per year on average, fog deposition increases the amount of water available to plants during austral winter, resuling in green southern-facing mountain slopes along the coastal strip of Peru. 
 The fog, which develops below the temperature inversion caused by the cold Humboldt current in austral winter, provides the name for this habitat.
-Every few years, the El Niño phenomenon brings torrential rainfall to this sun-baked environment [@dillon_lomas_2003].
-This causes the desert to bloom, and provides tree seedlings a chance to develop roots long enough to survive the following arid conditions.
+Every few years, the El Niño phenomenon brings torrential rainfall to this sun-baked environment, providing tree seedlings a chance to develop roots long enough to survive the following arid conditions [@dillon_lomas_2003].
 
-Unfortunately, fog oases are heavily endangered, primarily due to human activity (agriculture and climate change).
-To effectively protect the last remnants of this unique vegetation ecosystem, evidence is needed on the composition and spatial distribution of the native flora [@muenchow_predictive_2013; @muenchow_soil_2013].
-*Lomas* mountains also have economic value as a tourist destination, and can contribute to the well-being of local people via recreation.
-For example, most Peruvians live in the coastal desert, and *lomas* mountains are frequently the closest "green" destination.
-
-In this chapter we will demonstrate ecological applications of some of the techniques learned in the previous chapters.
-This case study will involve analyzing the composition and the spatial distribution of the vascular plants on the southern slope of Mt. Mongón, a *lomas* mountain near Casma on the central northern coast of Peru (Figure \@ref(fig:study-area-mongon)).
+Unfortunately, fog oases are heavily endangered, primarily due to agriculture and anthropogenic climate change.
+Evidence on the composition and spatial distribution of the native flora can support efforts to protect remaining fragments of fog oases [@muenchow_predictive_2013; @muenchow_soil_2013].
+The chapter also demonstrates how to apply techniques covered in previous chapters to an important applied field: ecology.
+In it you will analyze the composition and the spatial distribution of vascular plants (plants which have woody or 'lignified tissue' such as ferns) on the southern slope of Mt. Mongón, a *lomas* mountain near Casma on the central northern coast of Peru (Figure \@ref(fig:study-area-mongon)).
```

Comment:
Pls delete the part in the brackets because it is rather misleading (plants which have woody...)

Reply:
Deleted. Do you have a short definition for people who have never heard of vascular plants before?

Reply:
vascular plants (here mostly referring to flowering plants).

I would have to check but I guess, we only recorded /found flowering plants. So with saying 'mostly referring to' we are on the safe side.

Reply:
Or just say "plants" without the vascular bit. What value does it add to the reader? They can look up the details in the papers is my thinking..

Reply:
It tells the reader that we did not record mosses, lichens, mushrooms, etc.